framework/qlf/ui_channel/upstream.py

```diff
@@ -58,26 +64,49 @@ def get_camera_log(cam):
             log = arq.readlines()
             return log
 
-    except Exception as e:
-        print(e)
+    except Exception as err:
+        logger.error(err)
         return "Error"
```

Comment:
Here return `return ['get_pipeline_log Error']` so that it doesn't break `qlf-ui`.

Reply:
How is your configuration file?